docs/examples/volume-snapshot/backupconfiguration.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: stash.appscode.com/v1beta1
+kind: BackupConfiguration
+metadata:
+  name: deployments-volume-snapshot
+  namespace: demo
+spec:
+  schedule: "* * * * *"
```

Comment:
How do you know when to use restic vs VolumeSnapshot? Something like that?
```
spec:
 driver: volumeSnapshotter
``

Reply:
renamed "snapshotter" to "driver"